_posts/2023-11-24-kinesis-video-stream.markdown

```diff
@@ -0,0 +1,384 @@
+---
+layout: postv2
+font: serif
+title: "Home Security Hacks On AWS: Create your own 'Musical Concierge'"
+description: "Simplify the Video Streaming in near real-time with Amazon Kinesis Video Streams"
```

Comment:
suggest to remove "the"
description: "Simplify Video Streaming in near real-time with Amazon Kinesis Video Streams"